lib/Runtime/Library/JsBuiltIn/JsBuiltIn.js

```diff
@@ -415,4 +418,31 @@
         return A;
     });
 
+    platform.registerFunction(FunctionsEnum.ObjectFromEntries, function (iterable) {
+        // #sec-object.fromentries
+        "use strict";
+        if (this === null || this === undefined) {
+            __chakraLibrary.raiseNeedObject("Object.prototype.fromEntries");
```

Comment:
Is this required in the spec? I don't see language in the spec referencing the `this` of the function